docs/journal/01-cold-storage.md

```diff
@@ -76,14 +92,13 @@ In this configuration, devices post event data to a Service Bus topic. Worker ro
 
 - **Throughput.** A worker role has less overhead than a web role (*NEED SOMEONE TO VERIFY THIS, AND MORE DETAIL ON WHY*)
```

Comment:
I think that the difference between worker and web roles is not worth discussing here.